File: 강재연/programmers/P181841.js
// 문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181841
// 시간: 0.04 ms
// 메모리: 33.5 MB

function solution(str_list, ex) {
    let answer = '';
    str_list.forEach( str =>{
       if(!str.includes(ex)) answer += str;           
    });
    return answer;
}

File: 공예진/programmers/P181841.java
//문제: https://school.programmers.co.kr/learn/courses/30/lessons/181841
//시간: 0.06ms
//메모리: 72.2MB

class Solution {
    public String solution(String[] str_list, String ex) {
        String answer = "";
        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < str_list.length; i++) {
            if (!str_list[i].contains(ex)) {
                sb.append(str_list[i]);
            }
        }
        return sb.toString();
    }
}